CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Add Legacy LTI configuration XML
 - Add an instructor widget to modify join the discussion mode
 - Prevent join API calls in join mode denied
+- Add a widget for the instructor to upload a live video thumbnail
```

Comment:
Remember to move it in the `Unreleased` section

Reply:
Yes, ty !